### PR TITLE
UNIFY-642-cypress app

### PR DESCRIPTION
**Pull request by jaffrepaul**
https://cypress-io.atlassian.net/browse/UNIFY-642

**NOTE**- there are some `runner` options that we just need to be aware of as things that wouldn't change in this sort of update. e.g https://docs.cypress.io/api/commands/screenshot#Arguments

### CODE REVIEW

File: content/guides/references/configuration.md [hunk 1]
```diff
@@ -191,18 +191,17 @@ The Node version is used in Cypress to:
...
 
-<DocsImage src="/img/guides/test-runner-settings-nodejs-version.jpg" alt="Node version in Settings in Test Runner" ></DocsImage>
+<DocsImage src="/img/guides/test-runner-settings-nodejs-version.jpg" alt="Node version in Settings in Cypress App" ></DocsImage>
 
 ### Experiments
 
 Configuration might include experimental options currently being tested. See
 [Experiments](/guides/references/experiments) page.
 
-## Runner Specific Options
```
Comment (cowboy) [inline content/guides/references/configuration.md:201]: Anywhere you've added the text "Test Type" can you use "Testing Type" instead? This mirrors the `Cypress.testingType` property.

File: content/guides/references/configuration.md [hunk 2]
```diff
@@ -535,8 +536,8 @@ send a `503` status code. As a convenience it also sets a
 
 ### devServer / devServerConfig
 
-The `devServer` function is a required [`component`](#component) runner specific
-option, and allows you to register a component testing dev server. It receives a
+The `devServer` function is a required [`component`](#component) testing option,
```
Comment (cowboy) [inline content/guides/references/configuration.md:539]: Anywhere it said "runner specific option" can you say "testing specific option"? I feel the "specific" is important.

File: content/guides/references/error-messages.md [hunk 3]
```diff
@@ -851,9 +851,9 @@ one of the following:
 
 **You visit the Cypress proxy URL outside of a Cypress browser.**
 
-- Don't copy the URL you see when launching a Cypress browser from the Test
-  Runner and open it in a non-Cypress browser. If you want to run your tests in
-  a different browser, follow the instructions in the
+- Don't copy the URL you see when launching a Cypress browser from the
+  application and open it in a non-Cypress browser. If you want to run your
```
Comment (cowboy) [inline content/guides/references/error-messages.md:855]:
```suggestion
  Cypress App and open it in a non-Cypress browser. If you want to run your
```

File: content/guides/references/troubleshooting.md [hunk 4]
```diff
@@ -471,7 +471,7 @@ to learn about the many ways you can contribute.
 ### Run the Cypress app by itself
 
 Cypress comes with an npm CLI module that parses the arguments, starts the Xvfb
-server (if necessary), and then opens the Test Runner application built on top
+server (if necessary), and then opens the Cypress App application built on top
```
Comment (cowboy) [inline content/guides/references/troubleshooting.md:474]:
```suggestion
server (if necessary), and then opens the Cypress App built on top
```

File: content/guides/references/troubleshooting.md [hunk 5]
```diff
@@ -577,7 +577,7 @@ Calling _exit(1). Core file will not be generated.
 ### Patch Cypress
 
 Cypress comes with an npm CLI module that parses the arguments, starts the Xvfb
-server (if necessary), and then opens the Test Runner application built on top
+server (if necessary), and then opens the Cypress App application built on top
```
Comment (cowboy) [inline content/guides/references/troubleshooting.md:580]:
```suggestion
server (if necessary), and then opens the Cypress App built on top
```

File: content/guides/references/troubleshooting.md [hunk 6]
```diff
@@ -657,9 +657,9 @@ edit.
 
 You can change anything in the JavaScript code:
 
-<DocsImage src="/img/guides/source-code.png" alt="Source code of the Test Runner in a text editor" ></DocsImage>
+<DocsImage src="/img/guides/source-code.png" alt="Source code of the Cypress App in a text editor" ></DocsImage>
 
-When finished, if necessary, remove the edited Test Runner version and reinstall
+When finished, if necessary, remove the edited application version and reinstall
```
Comment (cowboy) [inline content/guides/references/troubleshooting.md:662]:
```suggestion
When finished, if necessary, remove the edited Cypress App version and reinstall
```

File: content/guides/tooling/code-coverage.md [hunk 7]
```diff
@@ -736,7 +736,7 @@ following repositories:
   shows coverage for a React App that uses TypeScript.
 - [bahmutov/cypress-and-jest](https://github.com/bahmutov/cypress-and-jest)
   shows how to run Jest unit tests and Cypress unit tests, collecting code
-  coverage from both test runners, and then produce a merged report.
+  coverage from both applications, and then produce a merged report.
```
Comment (cowboy) [inline content/guides/tooling/code-coverage.md:739]:
```suggestion
  coverage from both test runners, and then produce a merged report.
```

File: content/guides/tooling/typescript-support.md [hunk 8]
```diff
@@ -184,10 +184,11 @@ module.exports = (on, config) => {}
 ### Clashing types with Jest
 
 If you are using both Jest and Cypress in the same project, the TypeScript types
-registered globally by the two test runners can clash. For example, both Jest
-and Cypress provide the clashing types for the `describe` and `it` functions.
-Both Jest and Expect (bundled inside Cypress) provide the clashing types for the
-`expect` assertion, etc. There are two solutions to disentangle the types:
+registered globally by the two test applications can clash. For example, both
```
Comment (cowboy) [inline content/guides/tooling/typescript-support.md:187]: Please revert this change.

File: content/guides/tooling/visual-testing.md [hunk 9]
```diff
@@ -14,7 +14,7 @@ title: Visual Testing
 
 ## Functional vs visual testing
 
-Cypress is a _functional_ Test Runner. It drives the web application the way a
+Cypress is a _functional_ application. It drives the web application the way a
```
Comment (cowboy) [inline content/guides/tooling/visual-testing.md:17]:
```suggestion
Cypress is a _functional_ test runner. It drives the web application the way a
```

**Review by cowboy — CHANGES_REQUESTED** [review]

File: content/guides/component-testing/introduction.md [hunk 10]
```diff
@@ -99,7 +99,7 @@ to create a new Webpack config, then continue following the
 ### ⚠️ Existing end-to-end users
 
 If you are using Cypress Component Testing in a project that also has tests
-written with the Cypress End-to-End test runner, you may want to configure some
+written with the Cypress End-to-End Cypress App, you may want to configure some
```
[the remark this answers is not in the document]
Comment (jaffrepaul) [reply content/guides/component-testing/introduction.md:102]: whoops that doesn't make sense

File: content/guides/references/configuration.md [hunk 11]
```diff
@@ -273,12 +272,14 @@ cypress run --config-file tests/cypress.config.js
...
 
-### Runner Specific Overrides
+### Testing Type-Specific Overrides
 
-In addition to setting [Runner Specific Options](#Runner-Specific-Options), you
-can override other configuration options for either the E2E or
-[Component Testing](/guides/overview/choosing-testing-type#What-is-Component-Testing)
-runners.
+In addition to setting
+[Testing Type-Specific options](#Test-Type-Specific-Options), you can override
```
Comment (cowboy) [inline content/guides/references/configuration.md:278]: Does `#Test-Type-Specific-Options` need to change?

**Review by cowboy — CHANGES_REQUESTED** [review]

**Review by cowboy — APPROVED** [review]
